ext/standard/proc_open.c

```diff
@@ -948,6 +966,36 @@ static zend_result set_proc_descriptor_from_resource(zval *resource, descriptors
 }
 
 #ifndef PHP_WIN32
+#if defined(USE_POSIX_SPAWN)
```

Comment:
Consider to indent nested preprocessor conditionals:
```suggestion
# if defined(USE_POSIX_SPAWN)
```
Analogous for the respective #else and #endif